generator/AWSPSGeneratorLib/HelpMaterials/Examples/DS/Remove-DSTrust.xml

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>Remove-DSTrust -TrustId (Get-DSTrust -DirectoryId d-123456ijkl).TrustId
```

Comment:
I don't have the infrastructure to test this, but this should also work
```
Get-DSTrust -DirectoryId d-123456ijkl -Select Trusts.TrustId | Remove-DSTrust
```

Reply:
Yes, It does work. Tested and corrected. 
`Get-DSTrust -DirectoryId d-123456ijkl -Select Trusts.TrustId | Remove-DSTrust`